packages/plugin-typescript/README.md

```diff
@@ -1,20 +1,12 @@
 # `@yarnpkg/plugin-typescript`
 
-This plugin automatically adds `@types/` packages into your dependencies when you add a package that's covered by one.
+This plugin automatically adds `@types/` packages into your dependencies when you add a package that doesn't include its own types.
```

Comment:
Currently, we don't check if there are types included. If there is a `@types/package` in the registry we install it. Ideally, it should work like your updated docs.

Reply:
Yep, I actually slightly anticipated the logic I'd like to have 😃 Whether types are present or not is in the Algolia index: https://github.com/algolia/npm-search/pull/346. So it would just be a matter of using that instead of checking the actual registry. Wanna try to PR it? 😋